templates/Signify/Factory/Models/TableBlock.ss

```diff
@@ -3,13 +3,11 @@
     <h2>$Title</h2>
   <% end_if %>
   <% if $TableDescription %>
-    <div class="table-block__description">
-      <% if $EnableRichLinks %>
-        $TableDescription.RichLinks
-      <% else %>
-        $TableDescription
-      <% end_if %>
-  </div>
+    <% if $EnableRichLinks %>
```

Comment:
Better.
It would be better still if we could take RichLinks out of the module entirely.